config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
When you create a validation, what would you expect to be returned? What is this Validation thing that you, as an API user, are creating? Where does this Validation, once created, exist?

I.e., is there a Validation as a Noun, not a Verb?

Rails routing is biased toward Resourceful CRUD, but that is not all Rails supports (http://guides.rubyonrails.org/routing.html#non-resourceful-routes). As such, this doesn't feel like an exception. It just isn't a Resource CRUD action, imo.

It seems to me you are shoehorning an action (validating) into a Resourceful shoe, and it just doesn't fit.